### PR TITLE
azure-cli:  Fixed filter criteria for resource list

### PR DESCRIPTION
**Pull request by yarunachalam**
``list only resource for specific resource group
Recently resource list delete failed
[Running test: 'az_net_vnet_del' and command : 'az network vnet delete --id /subscriptions/a5e130f6-1ae8-48f5-8ca3-322fa4d9800f/resourceGroups/oqaclirg11286882/providers/Microsoft.Network/virtualNetworks/oqaclirg11286882-vnet1']
(ResourceGroupNotFound) Resource group 'oqaclirg11286882' could not be found.
This is issue is due the filter criteria. Fixed across the tests. ``
- Verification run: WIP

### CODE REVIEW

File: data/publiccloud/azure_vn.sh [hunk 1]
```diff
@@ -311,7 +311,7 @@ cmd_status "az_configure_defaults_group" az configure --defaults group="${grpnam
 echo "List all resources for resource group ${grpname} { az resource list }"
 echo "*******************************************************************"
 
-cmd_status "az_resource_list" az resource list -o table
+cmd_status "az_resource_list " az resource list --name "${grpname}" -o table
```
Comment (ricardobranco777) [inline data/publiccloud/azure_vn.sh:314]:
```suggestion
cmd_status "az_resource_list" az resource list --name "${grpname}" -o table
```

File: data/publiccloud/azure_lb.sh [hunk 2]
```diff
@@ -242,38 +242,55 @@ echo "Deleting ..."
...
+            echo "vnet deleted ${rid}"
             ;;
       *"sshPublicKeys"*)
+            echo "Deleting publickey ${rid}"
             cmd_status "az_sshkey_del" az sshkey delete --id "${rid}" --yes
-            echo "publickey deleted " "${rid}"
+            echo "publickey deleted ${rid}"
             ;;
       *"availabilitySets".*)
+            echo "Deleting availabilty set ${rid}"
```
Comment (ricardobranco777) [inline data/publiccloud/azure_lb.sh:281]:
```suggestion
            echo "Deleting availability set ${rid}"
```
Small typo.

**Review by ricardobranco777 — APPROVED** [review]
Did you run `shellcheck` on it.  Just a couple of suggestions.

Otherwise, LGTM.

**Comment by pdostal** [comment]
Please update the title and description as well as the commit message:
 * Add "azure_cli" to PR title and commit message
 * The code in PR description should be formatted as code (```)

**Comment by pdostal** [comment]
I've also created #17250

**Comment by pdostal** [comment]
Can you please also move all the data related to `azure_more_cli` test module from `data/publiccloud/` to `data/publiccloud/azure_more_cli/`?

File: data/publiccloud/azure_lb.sh (hunk 2 above)
Comment (yarunachalam) [reply data/publiccloud/azure_lb.sh]: done

**Comment by pdostal** [comment]
Please also fix your system time so your commits are not "from the future".

**Review by pdostal — APPROVED** [review]
All good now :rocket:

**Pull request merged**